admin/index_m.html

```diff
@@ -211,6 +247,11 @@
                 settings.password = decrypt(secret, settings.password);
             }
 
+
+            if (settings.ip) {
+                settings.iporhost = settings.ip;
```

Comment:
did you missed to also the the name of the ip field in the form?

Reply:
You can read the old setting and set the value it to the new field or you hold the old field and set the value to the new setting -> both ways are correct
change is comming soon